app/views/home/index.html.erb

```diff
@@ -0,0 +1,57 @@
+    <nav class="navbar navbar-default navbar-expand-lg fixed-top">
+    <div class="container flex justify-content0-between align-items-center">
+
+      <span></span>
+     
+      <h6 class="title">CATEGORIES</h6>
+
+
+      <div></div>
+
+
+    </div>
+  </nav>
+<div class="d-flex justify-content-between align-items-center bg-white p-2">
+  <h6 class="pt-2">Categories</h6>
+
+  <%= link_to 'Add Category', new_group_path, class: 'btn btn-primary' %>
+</div>
+
+<div class="mb-5">
+
+<% if @groups.empty? %>
+
+    <div class="d-flex flex-column justify-content-center align-items-center m-5">
+
+        <p>Category is empty.</p>
+        <%= link_to 'Add Category', new_group_path, class: 'btn btn-primary' %>
+
+     </div>
+<% else %>
+
+
+ <% @groups.each do |group|%>
+     <%= link_to controller: 'groups', action: 'show', id: group.id do %>
+          <div class="d-flex justify-content-between align-items-center bg-white p-2 gap-6 m-2 shadow-sm" >
+
+            <div class="d-flex justify-content-start align-items-center gap-3">
+              <img src="<%= group.icon %>" height="100" width="100">
+              <div class="flex-column d-flex justify-content-between mt-4">
+                <h6 class="pb-3 text-black-60"><%= group.name.capitalize %></h6>
+                <p class="text-black-50"><%= group.created_at.strftime("%Y-%m-%d") %></p>
+              </div>
+            </div>
+            <div class="fw-bold">$100</div>
+        </div>
+      <% end %>
+    <% end %>
+<% end %>
+
+</div>
+  <div class="position-fixed bottom-0 w-100">
+    <div class="d-flex justify-content-center align-items-center bg-white p-2">
+
+  <%= link_to 'Add Category', new_group_path, class: 'btn btn-primary w-full w-50' %>
+</div>
+</div>
```

Comment:
- [ ] kindly try to match your page to the given wireframe
your app 
![Screenshot from 2023-10-20 16-55-40](https://github.com/SAZZAD1Q2/BudgetMate/assets/66097371/eb87c395-2d12-4d22-b7f4-14a511aadd2b)
given wireframe
![Screenshot from 2023-05-06 13-54-49](https://github.com/SAZZAD1Q2/BudgetMate/assets/66097371/c2424e4f-d2ce-499c-8616-3b69b14ba71a)